### DISCUSSION TITLE
Proposal - Create a Chinese-specific category

### DISCUSSION THREAD

**Discussion by tisonkun** [apache/streampark#2828] [Polls]
Now that we have a large amount of Chinese traffic, to improve their experience and prevent non-Chinese developers from being confused by those posts, I suggest we create a Chinese-specific category.

Concretely, a Q&A category named "中文" accepts questions only. Any open-ended discussion is, instead, required to be posted in English.

cc @wolfboys

**Comment by Al-assad** [comment]
Good idea!

**Comment by tisonkun** [comment]
I'm going to create the category and move all Chinese Q&A there on the weekend if there is no objection.

Following open-ended discussion will be asked to write in English.

**Comment by wolfboys** [comment]
Great proposal.
